src/api/api.js

```diff
@@ -1,10 +1,43 @@
 import axios from 'axios';
+const BASE_URL = 'https://hacker-news.firebaseio.com/v0/';
 
 /**
  *
- *
+ * @returns {object}
  * @param {*} storiesType
  */
-export const getStoriesIndex = storiesType => {
-  return axios.get(`https://hacker-news.firebaseio.com/v0/${storiesType}.json`);
+export const getStoriesIndexArray = storiesType => {
+  return axios
+    .get(`${BASE_URL}${storiesType}.json`)
+    .then(res => {
+      return res.data;
+    })
+    .catch(err => {
+      return err;
+    });
+};
+
+/**
+ *
+ * @returns {promise}
+ * @param {string} id
+ */
+export const getItem = id => {
```

Comment:
we use the term fetch when we make an api request to get the item. So, the function name `fetchItem` is preferrable.